### ISSUE TITLE
Timer doesn't work

### ISSUE THREAD

**Issue by froyglez** [SimpleMobileTools/Simple-Clock#452]
When I create a new timer and activate it, when the time expires it doesn't sound

**Comment by tibbi** [comment]
if you keep the app open, does it work?

**Comment by froyglez** [comment]
No, even with the app opened, it doesn't fire any sound, just a notification without sound

**Comment by tibbi** [comment]
well, check your device notifications settings or the app itself

**Comment by froyglez** [comment]
https://user-images.githubusercontent.com/60108777/235363471-9fee4997-b644-403c-ae9b-daeaf5a033bc.mp4

Thanks, before doing that, here is a video showing my problem

**Comment by froyglez** [comment]
I've checked the settings and I can't figure out how to solve it. I set a timer of 10 seconds, then I go out of the app and then I turn off the screen of my phone. After the 10 seconds have passed, it does not fire the alarm. But when I press the button to see my phone screen, the alarm starts. I don't know what settings I have to turn on so that, even when I'm out of the app or the screen is not turned on, after the time ends the alarm works.

**Comment by froyglez** [comment]
Hi, after checking my device notifications settings, now If I press the square button at the bottom of the screen and swipe the app window, the timer stops working, but if I lock my phone with the app opened, the timer works. What can I do so the app works even if I swipe the window app after pressing the bottom square button?
